incubator/elasticsearch/templates/NOTES.txt

```diff
@@ -4,7 +4,7 @@ Elasticsearch can be accessed:
 
   * Within your cluster, at the following DNS name at port 9200:
 
-    {{ template "elasticsearch.client.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
+    {{ template "elasticsearch.client.fullname" . }}.{{ .Release.Namespace }}.svc.{{ template "cluster.kubernetesDomain" . }}
```

Comment:
i think the config var for domain (and the helper) was removed in some PR some time ago.

Reply:
I will doublecheck. This is actually a resubmit from late June when that was the fix to access the ES cluster from within the cluster. I figured it would have failed the lint though if that were the case, but I will still doublecheck.

Reply:
Yeah seems it was in #7612. So is this working now, or is the fix something else?

Reply:
I think it will not be possible (an not needed?) to ouput the fqdn name - if there is no built-in way to get the k8s cluster domainname.

Reply:
WDYT? This can be closed?